Rocket.toml

```
@@ -1,3 +1,6 @@
+[global]
```

We should swap this from `global` to just scoped for `development`.

```suggestion
[development]
auth_key = "dummy_key"
```

Rocket's config will automatically detect the environment and retrieve the correct key from the correct env block.

For production, we'll attach that as an environment variable, of course.